openseeds/cohort-procedures-and-templates/speedblog.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: Speedblog guide
+image: https://images.unsplash.com/photo-1432821596592-e2c18b78144f
+photos:
+  name: RetroSupply
+  license: CC-BY
+  url: https://images.unsplash.com/photo-1432821596592-e2c18b78144f
+---
+
+# Speed Blog: Project name, team members
+
+-----
+
+_This document provides guidance for you to summarize your OLS journey so far and the project you worked on during the entire period of your cohort in a speedblog.  Please draft with your mentor, and share with the OLS team._
+
+----
+
+- Ideally your speed blog will be a written piece with a picture (max 2) and contain some links and references. In terms of word count you are aiming for between 500 and 1200 words.
+- We will post your speedblog on OLS website: openlifesci.org/posts
+- See this post for guidance on speed blogging: https://software.ac.uk/speed-blogging-and-tips-writing-speed-blog-post
```

Comment:
Please make this link clickable

Reply:
Okay, but the link leads to a deadend though (404).